nmoscommon/mdnsbridge.py

```diff
@@ -108,7 +108,12 @@ def _updateServices(self, srv_type):
             r = requests.get(req_url, timeout=0.5, proxies={'http': ''})
             if r is not None and r.status_code == 200:
                 # If any results, put them in self.services
-                self.services[srv_type] = r.json()["representation"]
+                self.services[srv_type] = []
+                for dns_data in r.json()["representation"]:
+                    if self.config["https_mode"] == "enabled" and dns_data["protocol"] == "https":
+                        self.services[srv_type].append(dns_data)
+                    elif self.config["https_mode"] != "enabled" and dns_data["protocol"] == "http":
+                        self.services[srv_type].append(dns_data)
```

Comment:
I feel like I'm having a deja vu, but this should probably log if neither branch is satisfied.